### ISSUE TITLE
Datepicker with TimePicker: screenreader doesn't inform about changed time

### ISSUE THREAD

**Issue by mrkshx** [primefaces/primeng#17455]
### Describe the bug

When using time picker and changing the time the screen reader is not informed.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/datepicker#time

### Environment

primeng v19
also tested and failing on: primeng v18 & primeng v17

### Angular version

19.x.x

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Start Screenreader (I tested with NVDA)
2. Open Timepicker
3. Change Time with buttons
4. Screenreader will not read changed time

### Expected behavior

When changing the time screenreader should inform user about the changed time.
